Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,25 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (IsNullOrEmpty(array)) return;
+
+            bool isSorted;
+            do
+            {
+                isSorted = true;
```

Comment:
It's the opposite I think. Is it sorted at the start? No.

Reply:
I should have done the opposite.
I was about to lost my mind in this homework and I didn't pay attention, but now I refactored this part of code.